### ISSUE TITLE
Check action design consistency (Chess, Shogi, AnimalShogi)

### ISSUE THREAD

**Issue by sotetsuk** [sotetsuk/pgx#876]
Is it easy to recover the square info with same operation?

- Animal Shogi: `x, sq = jnp.int8(a // 12), jnp.int8(a % 12)`

**Comment by sotetsuk** [comment]
Seems `sq = label % boardsize` is natural? (considering `Go` and `Hex`)
